src/main/java/com/jcabi/ssh/SSH.java

```diff
@@ -208,6 +214,30 @@ public SSH(final String adr, final int prt,
         this.login = user;
         Validate.notEmpty(this.login, "user name can't be empty");
         this.key = priv;
+        this.passphrase = null;
+        this.port = prt;
+    }
+    
+    /**
+     * Constructor.
+     * @param adr IP address
+     * @param prt Port of server
+     * @param user Login
+     * @param priv Private SSH key
+     * @param passphrase Private SSH key pass phrase.
+     * @throws UnknownHostException If fails
+     * @checkstyle ParameterNumberCheck (6 lines)
+     */
+    public SSH(final String adr, final int prt, final String user,
```

Comment:
@guzzard this constructor is too big, think we should create some object that will handle preparation of those objects, also validation.